wiki/Installation.md

```diff
@@ -1,36 +1,70 @@
+
+If you're a framework author and want to use SwiftCurrent as a dependency, update your `Package.swift` file:
 
 ## Get the package
+
 Add the following line to the package dependencies in `Package.swift`:
+
 ```swift
-.package(url: "https://github.com/wwt/Workflow.git", .upToNextMajor(from: "3.0.0")),
+.package(url: "https://github.com/wwt/SwiftCurrent.git", .upToNextMajor(from: "3.0.0")),
 ```
 
 ## Get the correct product
+
 Add one one of the following products to your target dependencies.
-### If you want to use Workflow with UIKit:
+
+#### **If you want to use SwiftCurrent with UIKit**
+
 ```swift
-.product(name: "WorkflowUIKit", package: "Workflow")
+.product(name: "SwiftCurrent", package: "SwiftCurrent"),
+.product(name: "SwiftCurrent_UIKit", package: "SwiftCurrent_UIKit"")
 ```
+
+#### **Your import statement in this case will be**
+
+```swift
+import SwiftCurrent_UIKit
+```
+
 `WorkflowUIKit` will need to be built on a platform that supports UIKit, such as iOS or macOS with Catalyst.
 
-### If you want to use Workflow without UIKit:
+#### **If you want to use SwiftCurrent without UIKit**
+
 ```swift
-.product(name: "Workflow", package: "Workflow"),
+.product(name: "SwiftCurrent", package: "SwiftCurrent"),
 ```
-You will need to build your own [Orchestration Responders](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/OrchestrationResponder.html) for your domains.
+#### **Your import statement in this case will be**
+
+
+```swift
+import SwiftCurrent
+```
+
+You will need to build your own [Orchestration Responders](https://gitcdn.link/cdn/wwt/SwiftCurrent/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/OrchestrationResponder.html) for your domains.
 
 # CocoaPods
-Set up [CocoaPods](https://cocoapods.org/) for your project, then include Workflow in your dependencies by adding one of the following lines to your `Podfile`: 
 
-### If you want to use Workflow with UIKit:
+Set up [CocoaPods](https://cocoapods.org/) for your project, then include SwiftCurrent in your dependencies by adding one of the following lines to your `Podfile`:
+
+#### **If you want to use SwiftCurrent with UIKit**
+
 ```ruby
-pod 'DynamicWorkflow/UIKit'
+pod 'SwiftCurrent/UIKit'
 ```
 
-### If you want to use Workflow without UIKit:
+#### **If you want to use SwiftCurrent without UIKit**
+
 ```ruby
-pod 'DynamicWorkflow/Core'
+pod 'SwiftCurrent/Core'
+```
+
+#### **In both of these cases your import statement will be**
+
+```swift
+import SwiftCurrent
 ```
-You will need to build your own [Orchestration Responders](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/OrchestrationResponder.html) for your domains.
+
+You will need to build your own [Orchestration Responders](https://gitcdn.link/cdn/wwt/SwiftCurrent/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/OrchestrationResponder.html) for your domains.
```

Comment:
This is linked to `SwiftCurrent/Core` and needs to be colocated with it or have a callback to if you used Core.

Reply:
Made some changes to this verbiage